Jenkinsfile

```
@@ -51,26 +51,26 @@ pipeline {
                 script {
                     if (env.BRANCH_NAME in dockerTags) {
                         withCredentials([usernamePassword(credentialsId: 'nexus-soramitsu-rw', usernameVariable: 'DOCKER_REGISTRY_USERNAME', passwordVariable: 'DOCKER_REGISTRY_PASSWORD')]) {
-                            env.DOCKER_REGISTRY_URL = "https://nexus.iroha.tech:19004"
+                            env.DOCKER_REGISTRY_URL = "https://docker.soramitsu.co.jp/soramitsu"
                             env.TAG = dockerTags[env.BRANCH_NAME]
                             sh "./gradlew dockerPush"
                         }
                         withCredentials([usernamePassword(credentialsId: 'nexus-nbc-deploy', usernameVariable: 'DOCKER_REGISTRY_USERNAME', passwordVariable: 'DOCKER_REGISTRY_PASSWORD')]) {
-                            env.DOCKER_REGISTRY_URL = "https://nexus.iroha.tech:19000"
+                            env.DOCKER_REGISTRY_URL = "https://docker.soramitsu.co.jp/bakong"
```

```suggestion
                            env.DOCKER_REGISTRY_URL = "https://nexus.iroha.tech:19000"
```
 `nexus.iroha.tech:19000` used by NBC they do not have access to `docker.soramitsu.co.jp`, we will move 19000 separately we should be careful it may delay the delivery hotfix to production.